### PR TITLE
Add guide for using purestake-api (algodv2)

### PR DESCRIPTION
**Pull request by ratik21**

### CODE REVIEW

File: packages/algob/src/internal/deployer_cfg.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ export class DeployerConfig {
     this.runtimeEnv = runtimeEnv;
     this.cpData = loadCheckpointsRecursive();
     this.algoOp = algoOp;
-    this.accounts = mkAccountIndex(runtimeEnv.network.config.accounts);
+    this.accounts = mkAccountIndex(runtimeEnv.network.config.accounts ?? []);
```
Comment (amityadav0) [inline packages/algob/src/internal/deployer_cfg.ts:38]: do we not need accounts in purestake api ?

File: docs/guide/purestake-api.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+};
+
+module.exports = {
+  networks: {
+    default: defaultCfg,
+    purestake: purestakeTestNetCfg
+  }
+};
+```
+
```
Comment (amityadav0) [inline docs/guide/purestake-api.md:39]: could you please add an example call using api ?

File: packages/algob/src/internal/deployer_cfg.ts (hunk 1 above)
Comment (ratik21) [reply packages/algob/src/internal/deployer_cfg.ts:38]: This is to handle a case if no accounts are passed in config.

File: docs/guide/purestake-api.md (hunk 2 above)
Comment (ratik21) [reply docs/guide/purestake-api.md:39]: The usage is same as we use `algodClient` in scripts. We just demonstrate how to add/switch to this network (originally users were not aware how to use algob with purestake)

**Review by amityadav0 — APPROVED** [review]

File: docs/guide/purestake-api.md (hunk 2 above)
Comment (amityadav0) [reply docs/guide/purestake-api.md:39]: could you please add a `accounts` variable it could be blank `accounts: []` here ? It would be good for users

Comment (ratik21) [reply docs/guide/purestake-api.md:39]: yes

**Pull request merged**